docs/open-webui-integration.md

```
@@ -0,0 +1,603 @@
+    bool(metadata.get('session_id'))
+    and metadata.get('params', {}).get('function_calling') != 'legacy'
+    and (model.get('info', {}).get('meta', {}).get('capabilities') or {}).get('builtin_tools', True)
+)
+...
+if use_builtin_tools:
+    form_data['messages'] = await add_file_context(...)   # prepends <attached_files>
+```
+
+So with a file attached, an agent that expected `what is this pdf about?` receives:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)